cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -292,7 +292,17 @@ func Run(clientID string, imageName string, dataDisk bool, osID string, customTr
 		oauth, ce, gcsLogsDisabled, cloudLogsDisabled, stdoutLogsDisabled, kmsKey, kmsKeyring,
 		kmsLocation, kmsProject, noExternalIP, userLabels, storageLocation, uefiCompatible); err != nil {
 
+		postProcessErrorForNetwork(err, network, w)
+
 		return w, err
 	}
 	return w, nil
 }
+
+func postProcessErrorForNetwork(err error, network string, w *daisy.Workflow) {
+	if derr, ok := err.(daisy.DError); ok {
+		if derr.ErrType() == "networkResourceDoesNotExist" && network == "" {
```

Comment:
Consider extracting as a constant in error.go.

Is the ErrType actually returning a lower case error name? Other errors (like resourceDNEError from error.go) are upper case.

Reply:
This is constructed by resource type name + resourceDNEError. All resource types have their own name.  That's why I didn't list all error names in error.go.
The purpose is to determine whether it's caused by network resource.